### PR TITLE
Switching color theme globally

### LINKED ISSUE #715: Enable active theme in Preferences and Cluster Settings
Switching between themes in `Preferences` could have immediate effect not only on cluster dashboard but in `Preferences` itself as well as `Cluster Settings` and left menu.

preferences
settings

### LINKED ISSUE #720: Light theme support for global pages
Cluster pages supports white theme. We should add now support for light theme for global pages (cluster settings, preferrences etc) too.

### PR DESCRIPTION
**Pull request by aleksfront**
- It spreads active theme on settings pages (Preferences, Cluster Settings, Workspaces, Add Cluster) and Cluster Menu
- Fixes some light theme issues

theme switching
add cluster
light theme
workspaces
settings

Fixes #715 
Fixes #720

### CODE REVIEW

**Comment by miskun** [comment]
Just commenting visuals: OK by me. Need code review still.

File: src/renderer/components/ace-editor/ace-editor.scss [hunk 1]
```diff
@@ -5,16 +5,19 @@
...
+    border-radius: $radius;
 
-  &.light {
     .ace_scrollbar {
-      @include custom-scrollbar($theme: dark);
+      @include custom-scrollbar(dark);
     }
   }
 
+  &.hidden {
```
Comment (ixrock) [inline src/renderer/components/ace-editor/ace-editor.scss]: Why this is should be hidden in CSS and not just dropped in JSX? Is there any special meaning?

File: src/renderer/components/app.scss [hunk 2]
```diff
@@ -31,6 +29,7 @@ html {
 
 html, body {
   height: 100%;
+  overflow: hidden;
```
Comment (ixrock) [inline src/renderer/components/app.scss:32]: Why this is needed?

File: src/renderer/components/layout/main-layout.scss [hunk 3]
```diff
@@ -69,6 +62,11 @@
     @include custom-scrollbar;
     $spacing: $margin * 2;
 
+    .theme-light & {
+      @include custom-scrollbar(dark);
+      overflow-y: scroll;
```
Comment (ixrock) [inline src/renderer/components/layout/main-layout.scss]: Is it needed only for `light` theme? If not why assigned here?

**Review by ixrock — APPROVED** [review]
LGTM

**Review by Nokel81 — APPROVED** [review]

File: src/renderer/components/app.scss (hunk 2 above)
Comment (aleksfront) [reply src/renderer/components/app.scss:32]: Preventing drawer to expand body size = horizontal scrollbar flickering.

Comment (ixrock) [reply src/renderer/components/app.scss:32]: ok, next time better to leave a comment in the code describing unusual behaviour :)

**Pull request merged**